controllers/auth.js

```diff
@@ -2,8 +2,9 @@ const passport = require("passport");
 const users = require("../models/users");
 const QrCodeAuthModel = require("../models/qrCodeAuth");
 const authService = require("../services/authService");
-const { SOMETHING_WENT_WRONG, DATA_ADDED_SUCCESSFULLY, BAD_REQUEST } = require("../constants/errorMessages");
-
+const dataAccess = require("../services/dataAccessLayer");
+const { SOMETHING_WENT_WRONG, DATA_ADDED_SUCCESSFULLY } = require("../constants/errorMessages");
+const USER_DOES_NOT_EXIST_ERROR = "User does not exist!";
```

Comment:
since other constants are being imported from `../constants/errorMessages` would be better if we create this constant in that file as well.

Reply:
we can do that, for this request itself, lets not extend it to other requests in this, hop this is fine?